r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -155,6 +179,71 @@ test_that("strptime", {
   # RE2 library (not available on Windows with R 3.6)
   skip_if_not_available("re2")
 
+  expect_equal(
```

Comment:
I would move the `skip_on_cran()` higher, like to here, and note that it's because strptime token support is not consistent across platforms.

Reply:
also are these all inside the same `test_that("strptime", ...`? can we break them into separate `test_that` calls with more descriptive names, to help with future debugging/readability?

Reply:
I moved `skip_on_cran()` up and split the relevant `strptime` test block into three.